### PR TITLE
wolfmqtt 1.4 (new formula)

### PR DESCRIPTION
**Pull request by phedoreanu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by phedoreanu** [comment]
```Error Message
failed: brew linkage --test wolfmqtt
Stacktrace
        Broken dependencies:
  /usr/local/opt/wolfssl/lib/libwolfssl.24.dylib (wolfssl)

All the audits passed without any warning. Any ideas?

File: Formula/wolfmqtt.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ]
+
+    system "./autogen.sh"
+    system "./configure", *args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system bin/"wolfmqtt-config", "--cflags", "--libs", "--prefix"
```
Comment (SMillerDev) [inline Formula/wolfmqtt.rb:36]: Please check some output here and possibly run a test that's more in line with what users are expected to run.

**Comment by SMillerDev** [comment]
Is this the final iteration of this formula?

**Comment by phedoreanu** [comment]
Yes.

**Comment by SMillerDev** [comment]
Thanks @phedoreanu! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by phedoreanu** [comment]
wtf @SMillerDev ?!

**Comment by SMillerDev** [comment]
It got merged, github is just confused by the homebrew workflow.

**Pull request closed without merging**